### DISCUSSION TITLE
Through Terratest want to write test cases on Azure Role assignment and DNS zone check

### DISCUSSION THREAD

**Discussion by pravinjec** [gruntwork-io/knowledge-base#660] [How do I]
Currently writing test cases in go with the use of terratest to perform testing on Azure Infrastructure including Kubernetes, docker and helm.

I want to add test cases for Role assignment and DNS zone check, and I am not able to find any plugin or any test case criteria so that i can add the test case for it, can anyone please guide me in finding the correct module and writing test cases for these 2 tasks?

---

<ins datetime="2023-02-07T10:42:19Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109875">Tracked in ticket #109875</a></p>
</ins>

**Comment by pravinjec** [comment]
@gruntwork-discussions I am not able to open this [Tracked in ticket #109875](https://support.gruntwork.io/hc/requests/109875)

Can you please mention the same here

**Comment by gitsstewart** [comment]
@pravinjec we don't currently have support for Azure in terratest. Although we'd like to eventually support more than AWS, it's not currently on our roadmap. Following the same design pattern for other platforms that Terraform supports is certainly possible, but as you can imagine, it's a lot of work. We want to make sure we're successful with AWS before spreading out to other platforms.
